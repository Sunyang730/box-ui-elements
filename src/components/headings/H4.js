// @flow
import * as React from 'react';
import classNames from 'classnames';

import './H4.scss';

type Props = {
    children: React.Node,
    className?: string,
};

const H4 = ({ children, className }: Props) => <h4 className={classNames('bdl-H4', className)}>{children}</h4>;

export default H4;
