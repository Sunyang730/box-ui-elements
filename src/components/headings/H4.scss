@import '../../styles/variables';

h4.bdl-H4 {
    @include font-family;

    color: $bdl-gray;
    font-size: 15px;
    font-weight: bold;
    letter-spacing: .3px;
    line-height: 22px;
}
