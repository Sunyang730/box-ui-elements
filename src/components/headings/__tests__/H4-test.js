// @flow
import * as React from 'react';

import H4 from '../H4';

describe('components/headings/H4', () => {
    const getWrapper = (props = {}) => shallow(<H4 {...props}>foo</H4>);

    test('should render H4', () => {
        const wrapper = getWrapper({ className: 'moo' });

        expect(wrapper).toMatchSnapshot();
    });
});
