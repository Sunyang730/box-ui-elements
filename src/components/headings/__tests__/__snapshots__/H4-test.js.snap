// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/headings/H4 should render H4 1`] = `
<h4
  className="bdl-H4 moo"
>
  foo
</h4>
`;
